src/components/Node.tsx

```diff
@@ -55,15 +52,12 @@ export class UnconnectedNode extends React.Component<IExtraProps & INode & INode
             return {
                 nodeID: dbid,
 
-                parent: -1,
+                parent: 1,
```

Comment:
This should be changed back to -1 . It is meant to be a nonsense value because the uninitialized node should never be used anyway, and getting a nonsense value error later will make it easier to detect..